src/Error/ExceptionRenderer.php

```diff
@@ -114,9 +125,13 @@ protected function _unwrap($exception)
      */
     protected function _getController()
     {
-        if (!$request = Router::getRequest(true)) {
-            $request = ServerRequestFactory::fromGlobals();
+        $request = $this->request;
```

Comment:
Using `$request = $this->request ?: Router::getRequest(true)` would avoid the nested `if`s.